spec/06-expressions.md

```diff
@@ -599,7 +599,7 @@ Evaluation of the block entails evaluation of its
 statement sequence, followed by an evaluation of the final expression
 ´e´, which defines the result of the block.
 
-A block expression `{´c_1´; ´\ldots´; ´c_n´; ´}` where ´s_1 , \ldots , s_n´ are
+A block expression `{´s_1´; ´\ldots´; ´s_n´; }` where ´s_1 , \ldots , s_n´ are
```

Comment:
The meaning is: if the `s_n` of the block are case clauses `c_n`. So `c` is intended. The extra quote must be a typo, but the final semicolon is also a typo.  Then the `s_1` should just be `c_1` etc. Note that this is not grammar syntax and the semicolons are also only for clarity.

Reply:
Oh, I got it. I didn't knew its actual meaning so made the first commit. I added a new commit, just removed the typo.